mlflow/pyfunc/cli.py

```diff
@@ -59,3 +59,4 @@ def predict(model_path, run_id, input_path, output_path):
     if output_path:
         out_stream = open(output_path, 'w')
```

Comment:
@xueyumusic, you should maybe use a `context manager` `with...:`. What do you think?